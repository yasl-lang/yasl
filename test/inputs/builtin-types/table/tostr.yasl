echo { .a:.A, 10: [.a, .b, .c, {}]}->tostr()
