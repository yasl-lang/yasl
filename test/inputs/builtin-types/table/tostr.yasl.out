{a: A, 10: [a, b, c, {}]}
